code/modules/projectiles/guncrafting/attachments/barrels.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/weapon/gun_attachment/barrel/short
+	name = "short barrel"
+	icon_state = "barrel_short"
+	range = 14
```

Comment:
I do and this makes disablers not mechanically better than tasers if you have good aim because tasers instastun and disablers stun in 3-4.